sdk/build.gradle

```diff
@@ -72,7 +72,7 @@ android {
 publish {
     def groupProjectID = 'com.karhoo'
     def artifactProjectID = 'nsdk-canary'
-    def publishVersionID = '1.0.35.26102020-1'
+    def publishVersionID = '1.1.0.29102020-1'
```

Comment:
I would suggest to call it 1.1.0.29102020-RC-1

Reply:
Alright, I will change and upload the version to the canary builds for testing.